ksql-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -3,6 +3,9 @@
     {
       "name": "stream stream left join",
       "format": ["AVRO", "JSON"],
+      "versions": {
```

Comment:
The pattern here is to mark the existing test, with its expected topologies for past versions, as only being valid up to the last release.  Then add a new test, that is run from 5.4 onwards, which has the new internal topic names.